src/components/forms/Phone/index.tsx

```diff
@@ -1,26 +1,49 @@
 import React from "react";
-import { TextField } from "@material-ui/core";
-import TextInput from "../TextInput";
+import MuiPhoneNumber from 'material-ui-phone-number';
 
 export default function Phone(props) {
   const {label, required, disabled, value, validatemessage, status, onChange, onBlur, readOnly} = props;
-
   if (readOnly) {
-    return ( <TextInput {...props} /> );
+    const disableDropdown = true;
+    return (
+      <div>
+        <MuiPhoneNumber
+        fullWidth
+        helperText={validatemessage}
+        placeholder=""
+        size="small"
+        required={required}
+        disabled={disabled}
+        onChange={onChange}
+        error={status === "error"}
+        label={label}
+        value={value}
+        InputProps={{
+            readOnly: true,
+        }}
+        disableDropdown={disableDropdown}
+      />
+      </div>
+    );
+  }
+
+  const handleChange = (inputVal) => {
+    let value = inputVal && inputVal.replace(/\D+/g, '');
+    value = `+${value}`;
+    onChange({value});
   }
 
   return (
-      <TextField
-        type="tel"
+      <MuiPhoneNumber
         fullWidth
         variant="outlined"
         helperText={validatemessage}
         placeholder=""
         size="small"
+        defaultCountry={'us'}
         required={required}
         disabled={disabled}
-        onChange={onChange}
-        onBlur={onBlur}
```

Comment:
I'm assuming that there's no impact on how the component works by removing onBlur. If you tab out of the component, does validation, etc. still work as expected?

Reply:
yes John, after removing onBlur also validations and all working as expected.